README.md

```diff
@@ -306,7 +319,12 @@ More [`examples`](#examples)
 
 #### `globOptions`
 
-Type: `Object`
+Type:
+
+```ts
+type globOptions = object;
```

Comment:
Let's use types from `globby`, just use the same name

Reply:
import("globby").Options?